File: CP website/Leetcode/1281.subtract_the_product_and_sum_of_digits_of_an_integer.py
# https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/
class Solution:
    def subtractProductAndSum(self, n: int) -> int:
        
        
        s = 0
        p = 1
        
        while n:
            r = n%10
            s+=r
            p*=r
            
            n//=10
            
        return p-s

File: CP website/Leetcode/1281. Subtract the Product and Sum of Digits of an Integer.java
// https://leetcode.com/problems/subtract-the-product-and-sum-of-digits-of-an-integer/
class Solution {
    public int subtractProductAndSum(int n) {
     
         int pro = 1;
        int s = 0;
        while(n>0){
            int rem = n%10;
            pro = pro * rem;
            s = s + rem;
            n = n/10;
            
        }
        return pro-s;
    }
}